waiter/src/waiter/service.clj

```diff
@@ -43,15 +43,24 @@
 
 ;;; Service instance blacklisting, work-stealing, access and creation
 
+(defn instance-rpc-message
```

Comment:
Let's not move from positionality to more positionality.  Why not have the maps be inline?

Reply:
Done.